src/components/DeleteDialog.tsx

```diff
@@ -13,7 +13,7 @@ import {
   DialogDescription,
   DialogFooter,
 } from './ui/dialog';
-import { ArchiveRestore, Trash } from 'lucide-react';
+import { ArchiveRestore, Trash2 } from 'lucide-react';
```

Comment:
use icon file we have and import it from there, do the favour for archiveRestore too